### DISCUSSION TITLE
Clarification on New Tracks Behavior/Docs

### DISCUSSION THREAD

**Discussion by jmdelahanty** [talmolab/sleap#778] [Help!] [answered]
Hello!

One member of my lab told me today about how they are experiencing an issue where predictions are gathering many extra "tracks" over time. For example, if they have 3 mice in the video, they end up with several hundred "tracks" that they need to re-assign later.

They told me that @talmo mentioned using the `tracking.target_instance_count`, `tracking.post_connect_single_breaks`, and `tracking.clean_instance_count` parameters when running predictions, but it appears that the extra tracks aren't being pruned still. They also told me that their videos sometimes have mice that will climb out of the view of the camera on occasion and that it's probably related to that.

Here's an example command that they use in their shell script they gave me:

`sleap-track -m "models/path" -m "models/path" --tracking.tracker simple --tracking.track_window 4 --tracking.target_instance_count 4 --tracking.post_connect_single_breaks 1 --tracking.clean_instance_count 4 --tracking.clean_iou_threshold 0.7 --tracking.similarity iou --tracking.match greedy  -o "output/path.slp" "video/path.mp4"`

They also mentioned that the [docs](https://sleap.ai/guides/cli.html#sleap-track) for these parameters felt somewhat confusing to them.

**Comment by talmo** [comment, marked as answer]
We've recently updated the [Tracking and proofreading guide](https://sleap.ai/guides/proofreading.html) -- does this help to clarify?

**Reply by jmdelahanty** [reply]
I'll pass this along to them, I'm not sure they're aware of the guide yet.

**Reply by jmdelahanty** [reply]
As far as I can tell/have heard, the guide helps! Will use this as the answer for now and can report back if something else comes up.
